server/index.js

```diff
@@ -2,12 +2,19 @@ import express from 'express';
 import notes from './routes/notes.js';
 import auth from './routes/auth.js';
 import users from './routes/users.js';
+import cors from 'cors';
 
 import { configDotenv } from 'dotenv';
 
 configDotenv();
 const app = express();
 
+app.use(cors({
+  origin: 'http://localhost:3000', 
+  methods: ['GET', 'POST', 'PUT', 'DELETE'],
+  allowedHeaders: ['Content-Type', 'Authorization'] 
+}));
+
```

Comment:
If you've added this cors stuff, you've then left my manual cors header below it, which you can probably merge (for the development check) and delete.